### PR TITLE
Combined changes

### PR DESCRIPTION
**Pull request by zhangkungmail**
- Use lafaspot pdfbox 1.0.1
- Populate pdf timeout exceptions
- Remove call from tika-parsers to jempbox (it's only used to get metadata)
- Remove dependency on pdfbox-tools

### CODE REVIEW

**Comment by jessieZZZZ** [comment]
👍

File: tika-external/src/main/java/org/apache/tika/parser/pdf/AbstractPDF2XHTML.java [hunk 1]
```diff
@@ -328,8 +327,8 @@ void doOCROnCurrentPage() throws IOException, TikaException, SAXException {
             Path tmpFile = tmp.createTempFile();
             try (OutputStream os = Files.newOutputStream(tmpFile)) {
                 //TODO: get output format from TesseractConfig
-                ImageIOUtil.writeImage(image, config.getOcrImageFormatName(),
-                        os, config.getOcrDPI());
+                /*ImageIOUtil.writeImage(image, config.getOcrImageFormatName(),
```
Comment (ashutoshvsingh) [inline tika-external/src/main/java/org/apache/tika/parser/pdf/AbstractPDF2XHTML.java:330]: delete the code

File: tika-external/src/main/java/org/apache/tika/parser/pdf/PDF2XHTML.java [hunk 2]
```diff
@@ -306,7 +307,7 @@ private void writeToBuffer(PDImageXObject pdImage, String suffix, OutputStream o
                 org.apache.pdfbox.io.IOUtils.copy(data, out);
                 org.apache.pdfbox.io.IOUtils.closeQuietly(data);
             } else{
-                ImageIOUtil.writeImage(image, suffix, out);
+                //ImageIOUtil.writeImage(image, suffix, out);
```
Comment (ashutoshvsingh) [inline tika-external/src/main/java/org/apache/tika/parser/pdf/PDF2XHTML.java:310]: do not leave commented code

File: tika-parsers/src/main/java/org/apache/tika/parser/pdf/PDF2XHTMLPureJava.java [hunk 3]
```diff
@@ -306,7 +307,7 @@ private void writeToBuffer(PDImageXObject pdImage, String suffix, OutputStream o
                 org.apache.pdfbox.io.IOUtils.copy(data, out);
                 org.apache.pdfbox.io.IOUtils.closeQuietly(data);
             } else{
-                ImageIOUtil.writeImage(image, suffix, out);
+                //ImageIOUtil.writeImage(image, suffix, out);
```
Comment (ashutoshvsingh) [inline tika-parsers/src/main/java/org/apache/tika/parser/pdf/PDF2XHTMLPureJava.java:310]: remove commented code

File: tika-parsers/src/main/java/org/apache/tika/parser/pdf/PDFPureJavaParser.java [hunk 4]
```diff
@@ -214,13 +218,13 @@ private void extractMetadata(PDDocument document, Metadata metadata, ParseContex
...
 
-        if (xmp != null) {
+        /*if (xmp != null) {
             try {
                 dcSchema = xmp.getDublinCoreSchema();
             } catch (IOException e) {}
 
             JempboxExtractor.extractXMPMM(xmp, metadata);
-        }
+        }*/
```
Comment (ashutoshvsingh) [inline tika-parsers/src/main/java/org/apache/tika/parser/pdf/PDFPureJavaParser.java:227]: remove commented code

**Comment by ashutoshvsingh** [comment]
👍

**Pull request merged**